crates/frontend/src/contextmenu/mod.rs

```diff
@@ -1,17 +1,131 @@
 use std::rc::Rc;
-use futures_signals::signal::Mutable;
-
-pub struct ContextMenuState {
-    pub show_menu: Mutable<bool>,
-    pub menu_position: Mutable<(i32, i32)>,
+use dominator::{Dom, html, clone, events};
+use crate::{vfs::{Directory, File}, DEFAULT_DIRECTORY_MODE, DEFAULT_FILE_MODE};
+#[allow(dead_code)]
```

Comment:
Sorry I just saw this, please don't add things like this which disable warnings and linting.